packages/common-integration-tests/src/auth-client.ts

```diff
@@ -253,7 +253,7 @@ export function runAuthClientTests(
       const generateSuccessRst = generateResponse as GenerateApiKey.Success;
 
       // Wait 1sec for the token to expire
```

Comment:
nit: might want to update the comment here.

Reply:
ah thanks, fixed - https://github.com/momentohq/client-sdk-javascript/pull/1196/commits/c74c7d260c352880d1d09e2b42d9edc84328270b